beesdoo_worker_status_shift_swap/models/task.py

```diff
@@ -14,11 +14,7 @@ def _get_counter_date_state_change(self, new_state):
             and new_state in ["done", "absent_0"]
             and self.solidarity_offer_ids[0]
         ):
-            data["sr"] = 0
```

Comment:
Are you sure to delete this line ?

Reply:
Yes, setting the status to None prevents the counter update, so there's no need to change the counter manually. It is better than just setting data["sr"] to 0 because it also prevents the update of the next alert date, which was not the case before this commit.